### ISSUE TITLE
Can't change cell type to anything else than `code`.

### ISSUE THREAD

**Issue by lgautier** [jupyterlab/jupyterlab#10538]
## Description

The description is pretty much identical to the one for issue #2995. It seems like a return of it.

The selection menu for {code, Markdown, raw} appears broken. Either the selection menu disappears instantly (meaning that clicking on `code` displays only very briefly the drop-down with the other options, making the selection of anything impossible), or its stays long enough to allow a selection but the selection has not effect (meaning that selecting, say, Markdown will make the drop-cccckliblkgethichuhtbuiruefbfdnlrcvnuhdibb
down menu disappear but there is no effect and the cell is list marked as "code").

I have `browser.tabs.remote.autostart = true` (I can also see a `browser.tabs.remote.autostart.2 = true`).
`about:support` lists "Multiprocess Windows" set to `3/3`.

## Reproduce

The issue is present most of the time, although I occasionally have the cell selection work properly.
I have tried to reload the page (shift+reload), terminating and restarting (note I am restoring tabs when restarting Firefox), and restarting the computer. I can't seem to find a reproducible way to make the cell type selection work. It rarely works, its is unpredictable when it does, and when it does it does not last long.

## Expected behavior

The drop-down menu to select cell types should work.

## Context

- Operating System and version: Ubuntu 20.04.2
- Browser and version: Firefox 89.0.2
- JupyterLab version: 3.0.16

Both Firefox and jupyter are local

<details><summary>Troubleshoot Output</summary>
<pre>
$PATH:
	/usr/local/packages/R/4.0/bin
	/opt/software/python/py38_env/bin
	/usr/local/sbin
	/usr/local/bin
	/usr/sbin
	/usr/bin
	/sbin
	/bin
	/usr/games
	/usr/local/games
	/snap/bin

sys.path:
	/opt/software/python/py38_env/bin
	/usr/lib/python38.zip
	/usr/lib/python3.8
	/usr/lib/python3.8/lib-dynload
	/opt/software/python/py38_env/lib/python3.8/site-packages
	/opt/software/python/py38_env/lib/python3.8/site-packages/tzlocal-3.0b1-py3.8.egg
	/opt/software/python/py38_env/lib/python3.8/site-packages/pytz-2020.1-py3.8.egg
	/opt/software/python/py38_env/lib/python3.8/site-packages/backports.zoneinfo-0.2.1-py3.8-linux-x86_64.egg
	/opt/software/python/rpy2_github/.eggs/cffi-1.14.3-py3.8-linux-x86_64.egg
	/opt/software/python/rpy2_github/.eggs/pycparser-2.20-py3.8.egg
	/opt/software/python/py38_env/lib/python3.8/site-packages/rpy2-3.5.0.dev0-py3.8-linux-x86_64.egg
	/opt/software/python/py38_env/lib/python3.8/site-packages/rpy2_arrow-0.0.3-py3.8.egg

sys.executable:
	/opt/software/python/py38_env/bin/python3

sys.version:
	3.8.5 (default, May 27 2021, 13:30:53) 
	[GCC 9.3.0]

platform.platform():
	Linux-5.8.0-59-generic-x86_64-with-glibc2.29

which -a jupyter:
	/opt/software/python/py38_env/bin/jupyter

pip list:
	Package                                    Version     
	------------------------------------------ ------------
	adal                                       1.2.5       
	alabaster                                  0.7.12      
	altair                                     4.1.0       
	anyio                                      2.0.2       
	argon2-cffi                                20.1.0      
	astor                                      0.8.1       
	async-generator                            1.10        
	attrs                                      20.2.0      
	azure-cognitiveservices-search-imagesearch 2.0.0       
	azure-common                               1.1.25      
	Babel                                      2.8.0       
	backcall                                   0.2.0       
	backports.zoneinfo                         0.2.1       
	base58                                     2.1.0       
	bleach                                     3.2.1       
	blinker                                    1.4         
	blis                                       0.4.1       
	Brotli                                     1.0.9       
	cachetools                                 4.2.0       
	catalogue                                  2.0.4       
	certifi                                    2020.6.20   
	cffi                                       1.14.3      
	chardet                                    3.0.4       
	click                                      7.1.2       
	codecov                                    2.1.9       
	colorama                                   0.4.3       
	coverage                                   5.3         
	cryptography                               3.1.1       
	cycler                                     0.10.0      
	cymem                                      2.0.4       
	Cython                                     0.29.21     
	dash                                       1.19.0      
	dash-core-components                       1.15.0      
	dash-html-components                       1.1.2       
	dash-renderer                              1.9.0       
	dash-table                                 4.11.2      
	dataclasses                                0.6         
	decorator                                  4.4.2       
	defusedxml                                 0.6.0       
	docutils                                   0.16        
	dowhy                                      0.5.1       
	entrypoints                                0.3         
	fastai                                     2.1.5       
	fastbook                                   0.0.14      
	fastcore                                   1.3.2       
	fastprogress                               1.0.0       
	flake8                                     3.8.3       
	Flask                                      1.1.2       
	Flask-Compress                             1.8.0       
	future                                     0.18.2      
	gitdb                                      4.0.5       
	GitPython                                  3.1.12      
	graphviz                                   0.14.2      
	idna                                       2.10        
	imagesize                                  1.2.0       
	iniconfig                                  1.0.1       
	ipykernel                                  5.3.4       
	ipython                                    7.22.0      
	ipython-genutils                           0.2.0       
	ipywidgets                                 7.5.1       
	isodate                                    0.6.0       
	itsdangerous                               1.1.0       
	jedi                                       0.17.2      
	jeepney                                    0.4.3       
	Jinja2                                     2.11.2      
	joblib                                     0.17.0      
	json5                                      0.9.5       
	jsonschema                                 3.2.0       
	jupyter                                    1.0.0       
	jupyter-client                             6.1.7       
	jupyter-console                            6.2.0       
	jupyter-core                               4.6.3       
	jupyter-packaging                          0.7.12      
	jupyter-server                             1.5.1       
	jupyterlab                                 3.0.16      
	jupyterlab-pygments                        0.1.2       
	jupyterlab-server                          2.3.0       
	keyring                                    21.4.0      
	kiwisolver                                 1.3.1       
	MarkupSafe                                 1.1.1       
	matplotlib                                 3.3.2       
	mccabe                                     0.6.1       
	mistune                                    0.8.4       
	more-itertools                             8.5.0       
	mpmath                                     1.1.0       
	msrest                                     0.6.19      
	msrestazure                                0.6.4       
	multipledispatch                           0.6.0       
	murmurhash                                 1.0.4       
	mypy                                       0.790       
	mypy-extensions                            0.4.3       
	nbclassic                                  0.2.5       
	nbclient                                   0.5.1       
	nbconvert                                  5.6.1       
	nbdev                                      1.1.5       
	nbformat                                   5.0.8       
	nest-asyncio                               1.4.2       
	networkx                                   2.5         
	notebook                                   6.1.5       
	notedown                                   1.5.1       
	numpy                                      1.19.5      
	oauthlib                                   3.1.0       
	packaging                                  20.4        
	pandas                                     1.3.0       
	pandoc-attributes                          0.1.7       
	pandocfilters                              1.4.3       
	parso                                      0.7.1       
	pathy                                      0.6.0       
	patsy                                      0.5.1       
	pexpect                                    4.8.0       
	pickleshare                                0.7.5       
	Pillow                                     8.0.1       
	pip                                        20.0.2      
	pkg-resources                              0.0.0       
	pkginfo                                    1.5.0.1     
	plac                                       1.1.3       
	plotly                                     4.14.3      
	pluggy                                     0.13.1      
	preshed                                    3.0.4       
	prometheus-client                          0.8.0       
	prompt-toolkit                             3.0.7       
	protobuf                                   3.14.0      
	ptyprocess                                 0.6.0       
	py                                         1.9.0       
	pyarrow                                    4.0.1       
	pycodestyle                                2.6.0       
	pycparser                                  2.20        
	pydantic                                   1.7.4       
	pydeck                                     0.5.0       
	pydot                                      1.4.1       
	pyflakes                                   2.2.0       
	Pygments                                   2.7.1       
	pygraphviz                                 1.6         
	PyJWT                                      1.7.1       
	pyparsing                                  2.4.7       
	pyrsistent                                 0.17.3      
	pytest                                     6.2.1       
	pytest-cov                                 2.10.1      
	python-dateutil                            2.8.1       
	pytz                                       2020.1      
	PyYAML                                     5.3.1       
	pyzmq                                      19.0.2      
	qtconsole                                  5.0.1       
	QtPy                                       1.9.0       
	readme-renderer                            26.0        
	requests                                   2.24.0      
	requests-oauthlib                          1.3.0       
	requests-toolbelt                          0.9.1       
	retrying                                   1.3.3       
	rfc3986                                    1.4.0       
	rpy2                                       3.4.3       
	rpy2-arrow                                 0.0.3       
	rpy2-Matrix                                0.0.2       
	rpy2-R6                                    0.0.1       
	scikit-learn                               0.24.2      
	scipy                                      1.5.2       
	SecretStorage                              3.1.2       
	Send2Trash                                 1.5.0       
	sentencepiece                              0.1.94      
	setuptools                                 44.0.0      
	six                                        1.15.0      
	smart-open                                 5.1.0       
	smmap                                      3.0.5       
	sniffio                                    1.2.0       
	snowballstemmer                            2.0.0       
	spacy                                      3.0.6       
	spacy-legacy                               3.0.6       
	Sphinx                                     4.0.2       
	sphinxcontrib-applehelp                    1.0.2       
	sphinxcontrib-devhelp                      1.0.2       
	sphinxcontrib-htmlhelp                     1.0.3       
	sphinxcontrib-jsmath                       1.0.1       
	sphinxcontrib-qthelp                       1.0.3       
	sphinxcontrib-serializinghtml              1.1.4       
	srsly                                      2.4.1       
	statsmodels                                0.12.2      
	streamlit                                  0.75.0      
	sympy                                      1.7.1       
	terminado                                  0.9.1       
	testpath                                   0.4.4       
	thinc                                      8.0.7       
	threadpoolctl                              2.1.0       
	toml                                       0.10.1      
	toolz                                      0.11.1      
	torch                                      1.9.0+cu111 
	torchaudio                                 0.9.0       
	torchvision                                0.10.0+cu111
	tornado                                    6.1         
	tqdm                                       4.50.0      
	traitlets                                  5.0.4       
	twine                                      3.2.0       
	typed-ast                                  1.4.1       
	typer                                      0.3.2       
	typing-extensions                          3.7.4.3     
	tzlocal                                    3.0b1       
	urllib3                                    1.25.10     
	validators                                 0.18.2      
	wasabi                                     0.8.2       
	watchdog                                   1.0.2       
	wcwidth                                    0.2.5       
	webencodings                               0.5.1       
	Werkzeug                                   1.0.1       
	wheel                                      0.35.1      
	widgetsnbextension                         3.5.1
</pre>
</details>

**Comment by krassowski** [comment]
Hi @lgautier nice seeing you here. We are tracking it in #10189. I just noticed that the #2995 issue did not get updated with references to #10189 and subsequent duplicates (probably because it got locked) so this is a fourth duplicate; I just un-locked that one and commented to point to the new issue.

I had a go at fixing it last weekend but I did not come up with a solution. I wonder if this was caused by some change in Firefox 89 because the first time we got the report was from 89b.
